content/blog/2017-12-08-rprofile-jenny-bryan.md

```diff
@@ -0,0 +1,95 @@
+---
+slug: rprofile-jenny-bryan
+title: ".rprofile: Jenny Bryan"
+date: 2017-12-08
+preface: "In this occasional series, we interview someone using a loosely defined set of interview questions for the purpose of “demystifying the creative and development processes of R community members”. This interview was conducted and prepared by Kelly O'Briant as part of an rOpenSci unconf17 project."
```

Comment:
@kellobri can you take out the quotation signs from “demystifying the ...."? for this and future ones?